SubmittingPatches-backports.rst

```diff
@@ -234,12 +234,26 @@ the backport branch with the cherry-picks in it.
 
 The script is located at ``src/script/ceph-backport.sh`` in the ``master``
 branch. Though there might be an older version of this script in a stable
-branch, do not use it. Only use the most recent version from master.
+branch, do not use it. Only use the most recent version from master and
+best not from your active repository. That said, copy the script to a
```

Comment:
Only use the most recent version from master. Instead of running directly from a checkout of the master branch, you will need to copy the script to a dedicated path . . .